src/store/store.ts

```diff
@@ -166,81 +166,83 @@ export const useStore = defineStore('store', {
       return this.getOriginalBusinessInfo?.nrNumber || ''
     },
 
-    // Original entity type getters
-    isOriginBcCompany (): boolean {
-      return (this.getOriginalLegalType === CorpTypeCd.BC_COMPANY)
-    },
-
-    isOriginBenefitCompany (): boolean {
-      return (this.getOriginalLegalType === CorpTypeCd.BENEFIT_COMPANY)
-    },
-
-    isOriginBcUlcCompany (): boolean {
-      return (this.getOriginalLegalType === CorpTypeCd.BC_ULC_COMPANY)
-    },
-
-    isOriginBcCcc (): boolean {
-      return (this.getOriginalLegalType === CorpTypeCd.BC_CCC)
-    },
-
     /** The entity type. */
     getEntityType (): CorpTypeCd {
       return this.stateModel.tombstone.entityType
     },
 
     /** Whether the entity is a Benefit Company. */
-    isBenefitCompany (): boolean {
+    isEntityBenefitCompany (): boolean {
       return (this.getEntityType === CorpTypeCd.BENEFIT_COMPANY)
     },
 
-    /** Whether the entity is a BC Corporation. */
-    isBcCorporation (): boolean {
-      return (this.getEntityType === CorpTypeCd.BC_CORPORATION)
-    },
```

Comment:
This is entity type "CR", which doesn't exist in LEAR. (It's actually a special Names-only type, that gets converted to BC in LEAR.)